### DISCUSSION TITLE
Download genome files not working?

### DISCUSSION THREAD

**Discussion by RonKeller** [kblin/ncbi-genome-download#230] [General]
dear community,

I attempted to download all available E coli genomes from the NCBI database using your package via this command:
ncbi-genome-download --species-taxids 562 bacteria

This finished without any problems, and yielded about 38 000 output folders in the subfolders refseq/bacteria/ .
However, when I checked those, most contained exactly one file 
MD5SUMS
which looks like this:
`201a7cdb8790811d81e20272b4f0dc0b  ./annotation_hashes.txt
e02ba9d6e803764a0f2fb6426f235dbd  ./GCF_017117745.1_ASM1711774v1_assembly_report.txt
3cb5e81c0fc96aa50d9e402cfedc68e9  ./GCF_017117745.1_ASM1711774v1_assembly_stats.txt
58a500b8e6d12ba89d7258c56eea3646  ./GCF_017117745.1_ASM1711774v1_cds_from_genomic.fna.gz
3d0342f149e9e799b6bb11aa67606af7  ./GCF_017117745.1_ASM1711774v1_feature_count.txt
0537f0fddec806fffe92683f2b50c8f0  ./GCF_017117745.1_ASM1711774v1_feature_table.txt.gz
614fa5a918f91e9d0348a4f3f646a647  ./GCF_017117745.1_ASM1711774v1_genomic.fna.gz
9a9f2f71a0ae8dc0355306aeef48d239  ./GCF_017117745.1_ASM1711774v1_genomic.gbff.gz
620f9ff372b7bd0a3514a04623ec967a  ./GCF_017117745.1_ASM1711774v1_genomic.gff.gz
5d1c171dcd2c8814180ef06da3273b1b  ./GCF_017117745.1_ASM1711774v1_genomic.gtf.gz
be0ba445b06b2ab2dfc183370a1f63fc  ./GCF_017117745.1_ASM1711774v1_protein.faa.gz
76a67055c935f89f231b449cf86fcb28  ./GCF_017117745.1_ASM1711774v1_protein.gpff.gz
ac3253b88edb24e979dd53706876517d  ./GCF_017117745.1_ASM1711774v1_rna_from_genomic.fna.gz
cbe26f962ecd9f7e6d1e0ed3afd369b3  ./GCF_017117745.1_ASM1711774v1_translated_cds.faa.gz
bb14c6511a80dbc60878b22521cb3a17  ./GCF_017117745.1_ASM1711774v1_wgsmaster.gbff.gz`
However, I cannot find the files mentioned here on my drive; if I check the folder contents, only 79 of the 38 000 folders contain more than one file, and that file is MD5SUMS.

What am I missing here? Those files should be in the same folder as the MD5 file, right?

Best
Ronald

**Comment by kblin** [comment]
Are you sure the command finished successfully? Because `ncbi-genome-download` will get the md5sum files first, and then download the sequence files. You could try running the command again, maybe with `--progress-bar` if you're running interactively.
